modules/creating-a-custom-training-image.adoc

```diff
@@ -0,0 +1,118 @@
+:_module-type: PROCEDURE
+
+[id='creating-a-custom-training-image_{context}']
+= Creating a custom training image
+
+You can create a custom training image by adding packages to a base training image.
+
+.Prerequisites
+
+* You have chosen a training image to use as the base for your custom image. 
+ifndef::upstream[]
++
+See link:https://access.redhat.com/articles/rhoai-supported-configs[Supported Configurations] for a list of the {productname-short} base training images and their preinstalled packages.
+endif::[]
+
+* You can access the base training image.
```

Comment:
maybe combine the second prerequisite with the first one - for example:
You have chosen-and can access-a training image to use as the base for your custom image. 
or
You have chosen a training image to use as the base for your custom image and you have access to the base training image.